.github/workflows/publish.yml

```diff
@@ -52,6 +52,31 @@ jobs:
         env:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      -
+        name: Update redirects
+        run: |
+          path_join() {
+            part1=$1
+            part2=$2
+            if [ "${part1: -1}" == "/" ]; then
```

Comment:
Shellcheck warns that `"${part1: -1}:` is non-POSIX; do we need to explicitly set `bash` as shell?

Reply:
Actually; I think `${part1%/}` would also work; basically: "normalise" by stripping the trailing `/` (if any), then append the `index.html`;

```bash
export foo="/hello/"
export bar="/hello"
echo "${foo%/}/index.html"
# /hello/index.html

echo "${bar%/}/index.html"
# /hello/index.html
```